icons/pug/pug-original.svg

```diff
@@ -0,0 +1 @@
cac8d"/><path id="path22" stroke-width="1" d="M78.624 55.5s-5.894-6.2-13.787-6.4h.2c-7.992.2-13.786 6.4-13.786 6.4 6.893-4.8 12.787-4.7 13.786-4.7-.1 0 6.694-.1 13.587 4.7z" fill="#ccac8d"/><path id="path23" stroke-width="1" d="M71.73 42.5s-2.996-4.2-6.992-4.3h.2c-2.998.1-6.894 4.3-6.894 4.3 3.397-3.3 6.893-3.2 6.893-3.2s3.297-.1 6.794 3.2z" fill="#ccac8d"/><path id="path24" stroke-width="1" d="M37.164 73.2s-4.695 2.3-8.092.9h-.1c-2.997-1.7-4.496-6.8-4.496-6.8s2.998 9 12.688 5.9z" fill="#ccac8d"/><path id="path25" stroke-width="1" d="M91.911 73.2s4.696 2.3 8.092.9c3.996-1.7 4.596-6.8 4.596-6.8s-2.997 9-12.688 5.9z" fill="#ccac8d"/><path id="path26" stroke-width="1" d="M42.559 41.2c2.597-.5 6.893-.6 10.29.5 4.296 1.5.8 7 1.698 7.3.9.3 2.098-3.8 10.09-3.4 8.093.4 8.992 4 10.09 3.4 1.1-.6-1.098-10 10.99-7.8 0 0-12.688-3.4-12.088 5.8 0 0-7.293-5.6-17.483-.6.1 0 2.697-8.6-13.587-5.2z" fill="#ccac8d"/><path id="path27" stroke-width="1" d="M86.816 41.2c.2 0 .3.1.4.1.1 0-.1-.1-.4-.1z" fill="#ccac8d"/><path id="path28" stroke-width="1" d="M86.816 41.2Z" fill="#ccac8d"/><path id="path29" stroke-width="1" d="M39.062 28.9s-10.79 13.6-12.388 18.8c-1.598 5.3-2.797 27-4.196 30.1l-4.995-21.4 9.191-22.3Z" fill="#ccac8d"/><path id="path30" stroke-width="1" d="M89.813 28.9s10.79 13.6 12.388 18.8c1.599 5.3 2.798 27 4.196 30.1l4.995-21.4-9.19-22.3z" fill="#ccac8d"/><path id="path31" stroke-width="1" d="M89.314 28.9s11.588 9.7 14.985 20.9c3.397 11.2 1.998 24.8 4.596 26.5 3.696 2.4 7.892-11.9 9.29-13.4 2.199-2.4 9.492-8.5 9.991-9.6.5-1.1-14.786-17.8-21.48-21.1-8.091-3.8-18.082-4.1-17.382-3.3Z" fill="#56332b"/><path id="path32" stroke-width="1" d="M99.204 34.9s13.687 17.5 13.487 39.3l5.495-11.2c-.1 0-4.896-14.3-18.982-28.1z" fill="#442823"/><path id="path33" stroke-width="1" d="M39.062 28.9s-11.588 9.7-14.985 20.9C20.68 61 22.079 74.6 19.48 76.3c-3.696 2.4-7.892-11.9-9.29-13.4C7.991 60.5.7 54.4.2 53.3c-.5-1.1 14.786-17.8 21.48-21.1 8.091-3.8 18.082-4.1 17.382-3.3Z" fill="#56332b"/><path id="path34" stroke-width="1" d="M29.172 34.9S15.485 52.4 15.685 74.2L10.29 63s4.796-14.3 18.882-28.1z" fill="#442823"/><path id="path35" stroke-width="1" d="M21.78 74.6s.998 5.4 2.597 7.1c1.598 1.7.5-1.3.5-1.3s-1.7-.9-1.4-7.8c.3-6.9-1.698 2-1.698 2z" fill="#ccac8d"/><path id="path36" stroke-width="1" d="M106.997 74.6s-1 5.4-2.598 7.1c-1.598 1.7-.5-1.3-.5-1.3s1.699-.9 1.4-7.8c-.3-6.9 1.698 2 1.698 2z" fill="#ccac8d"/><ellipse id="circle36" cx="54.447" cy="70.5" stroke-width="1" fill="#442823" rx=".799" ry=".8"/><ellipse id="circle37" cx="49.852" cy="75.3" stroke-width="1" fill="#442823" rx=".799" ry=".8"/><ellipse id="circle38" cx="48.353" cy="70.5" stroke-width="1" fill="#442823" rx=".799" ry=".8"/><ellipse id="circle39" cx="73.929" cy="70.5" stroke-width="1" fill="#442823" rx=".799" ry=".8"/><ellipse id="circle40" cx="78.524" cy="75.3" stroke-width="1" fill="#442823" rx=".799" ry=".8"/><ellipse id="circle41" cx="80.023" cy="70.5" stroke-width="1" fill="#442823" rx=".799" ry=".8"/></svg>
```

Comment:
This can be optimized quite a bit more. This version is about 6% smaller:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#efcca3" d="M107.296 50.9c-.2-4.4.4-8.3-1.598-11.6-4.796-8.2-16.784-13-40.76-13v.7h-.5.5v-.7c-23.977 0-36.565 4.8-41.36 13.1-1.899 3.4-1.699 7.2-1.999 11.6-.2 3.5-1.798 7.2-1.099 11.2.8 5.2 1.1 10.4 1.899 15.2.599 3.9 5.994 7.2 6.493 10.9 1.399 10.2 11.989 14.9 35.965 14.9v.8h-.599.7v-.8c23.976 0 34.166-4.7 35.465-14.9.5-3.8 5.495-7 6.094-10.9.8-4.8 1.099-10 1.898-15.2.7-4-.899-7.8-1.099-11.3z"/><path fill="#ccac8d" d="M64.538 54.5c4.296.1 7.293 2.8 10.09 5.3 3.297 2.9 8.891 4.9 11.19 7.4 2.297 2.5 5.294 5 6.393 8.9C93.31 80 93.61 85 93.61 86.3c0 1.3.699 1 2.697 0 4.695-2.3 9.89-8.5 9.89-8.5-.6 3.9-5.694 7.4-6.194 11.1C98.805 99.1 88.914 104 64.438 104h-.1.6"/><path fill="#ccac8d" d="M80.322 46.7c.9 3.1 4.096 13.6-2.098 10.1 0 0 2.598 1.5 4.196 7.2 1.699 5.7 5.795 6.4 5.795 6.4s6.693 1.3 11.688-3c4.196-3.6 4.896-10 3.097-14.9-1.798-4.8-4.995-6.3-9.69-7.3-4.696-1.1-14.087-2-12.988 1.5z"/><ellipse cx="92.211" cy="58.1" rx="8.791" ry="8.8"/><ellipse cx="89.913" cy="54.2" fill="#fff" rx="2.298" ry="2.3"/><path fill="#efcca3" d="M78.824 57.7s7.892 5.4 12.188 10.7c4.296 5.3 4.196 6.3 4.196 6.3l-3.097 1.4s-4.396-8.3-9.79-11.4c-5.495-3.1-6.095-5.7-6.095-5.7z"/><path fill="#ccac8d" d="M64.837 54.5c-4.295.1-7.492 2.8-10.39 5.3-3.296 2.9-9.09 4.9-11.389 7.4-2.297 2.5-5.394 5-6.493 8.9-1.1 3.9-1.499 8.9-1.499 10.2 0 1.3.2 1.4-2.697 0-4.696-2.2-9.89-8.5-9.89-8.5.599 3.9 5.694 7.4 6.193 11.1 1.4 10.2 11.29 15.1 35.766 15.1h.5"/><path fill="#56332b" d="M88.015 71.4C83.22 65.5 75.527 60 64.837 60h-.1c-10.69 0-18.382 5.5-23.177 11.4-4.995 6.1-4.596 8.5-4.596 14.3 0 21 7.393 15 12.288 17.6 4.996 2.5 10.19 1.7 15.486 1.7h.1c5.394 0 10.49.7 15.484-1.8 4.896-2.5 12.289 3.7 12.289-17.3.1-5.8.4-8.4-4.596-14.5z"/><path fill="#442823" d="M64.338 65.2s-.7 9.7-2.098 11.6l2.597-.6z"/><path fill="#442823" d="M65.037 65.2s.7 9.7 2.098 11.6l-2.597-.6z"/><path fill="#56332b" d="M56.645 62.9c-.999-2.3 2.598-6 8.292-6.1 5.695 0 9.291 3.7 8.292 6.1-.999 2.4-4.595 3.1-8.292 3.2-3.596-.1-7.293-.8-8.292-3.2Z"/><path d="M64.937 65.2c0-.4 3.397-.5 5.195-1.7 0 0-3.696 1.2-4.495.7-.8-.4-1-1.6-1-1.6s-.3 1.2-.899 1.6c-.699.4-4.895-.7-4.895-.7s5.595 1.4 5.595 1.7c0 .3-.1 1.3-.1 2 0 2.5 0 8.7.4 9.2.599.9.4-6.7.4-9.2-.1-.8-.1-1.6-.2-2z"/><path fill="#331712" d="M65.137 78.6c1.698 0 4.696 1.2 7.393 3.1-2.598-2.9-5.695-4.9-7.393-4.9-1.798 0-5.594 2.2-8.292 5.4 2.797-2.2 6.394-3.6 8.292-3.6z"/><path fill="#442823" d="M64.438 96.3c-3.797 0-7 493-1.2-10.89-2.1-.699-.2-1.398.3-2.098.1-6.294-2-11.389-5.4-14.486M54.947 85s-2.498 7.5-.8 10.8l-2.297-1s1.698-7.6 3.097-9.8zm19.781 0s2.497 7.5.8 10.8l2.297-1s-1.798-7.6-3.097-9.8z"/><path fill="#ccac8d" d="M48.553 46.7c-.899 3.1-4.096 13.6 2.098 10.1 0 0-2.597 1.5-4.196 7.2-1.598 5.7-5.794 6.4-5.794 6.4s-6.694 1.3-11.689-3c-4.196-3.6-4.895-10-3.097-14.9 1.798-4.9 4.995-6.3 9.69-7.3 4.696-1.1 13.987-2 12.988 1.5z"/><path d="M64.837 76.8c2.698 0 11.09 5.8 11.19 12.9v-.4c0-7.4-6.794-13.3-11.19-13.3-4.395 0-11.189 6-11.189 13.3v.4c.1-7.1 8.492-12.9 11.19-12.9z"/><ellipse cx="49.193" cy="76.207" fill="#331712" rx=".799" ry="1.5" transform="rotate(-14.474) skewX(-.027)"/><ellipse cx="77.742" cy="40.256" fill="#331712" rx=".799" ry="1.5" transform="rotate(17.251) skewX(.031)"/><ellipse cx="37.164" cy="58.1" rx="8.791" ry="8.8"/><ellipse cx="39.462" cy="54.2" fill="#fff" rx="2.298" ry="2.3"/><path fill="#331712" d="M67.435 58.2c0-.1-2.298 1-2.897 1.1-.6-.1-2.897-1.2-2.897-1.1h5.794z"/><path fill="#efcca3" d="M49.952 57.7s-7.893 5.4-12.188 10.7c-4.296 5.3-4.196 6.3-4.196 6.3l3.097 1.4s4.395-8.3 9.79-11.4C51.85 61.6 52.55 59 52.55 59z"/><path fill="#ccac8d" d="M32.669 41.7s-2.698 7.4-8.692 10.5c0 0 9.39-1.1 8.692-10.5zm63.039 0s2.697 7.4 8.691 10.5c0 0-9.39-1.1-8.691-10.5zM78.624 55.5s-5.894-6.2-13.787-6.4h.2c-7.992.2-13.786 6.4-13.786 6.4 6.893-4.8 12.787-4.7 13.786-4.7-.1 0 6.694-.1 13.587 4.7zm-6.894-13s-2.996-4.2-6.992-4.3h.2c-2.998.1-6.894 4.3-6.894 4.3 3.397-3.3 6.893-3.2 6.893-3.2s3.297-.1 6.794 3.2zM37.164 73.2s-4.695 2.3-8.092.9h-.1c-2.997-1.7-4.496-6.8-4.496-6.8s2.998 9 12.688 5.9zm54.747 0s4.696 2.3 8.092.9c3.996-1.7 4.596-6.8 4.596-6.8s-2.997 9-12.688 5.9z"/><path fill="#ccac8d" d="M42.559 41.2c2.597-.5 6.893-.6 10.29.5 4.296 1.5.8 7 1.698 7.3.9.3 2.098-3.8 10.09-3.4 8.093.4 8.992 4 10.09 3.4 1.1-.6-1.098-10 10.99-7.8 0 0-12.688-3.4-12.088 5.8 0 0-7.293-5.6-17.483-.6.1 0 2.697-8.6-13.587-5.2zm44.257 0c.2 0 .3.1.4.1.1 0-.1-.1-.4-.1zm0 0ZM39.062 28.9s-10.79 13.6-12.388 18.8c-1.598 5.3-2.797 27-4.196 30.1l-4.995-21.4 9.191-22.3Zm50.751 0s10.79 13.6 12.388 18.8c1.599 5.3 2.798 27 4.196 30.1l4.995-21.4-9.19-22.3z"/><path fill="#56332b" d="M89.314 28.9s11.588 9.7 14.985 20.9c3.397 11.2 1.998 24.8 4.596 26.5 3.696 2.4 7.892-11.9 9.29-13.4 2.199-2.4 9.492-8.5 9.991-9.6.5-1.1-14.786-17.8-21.48-21.1-8.091-3.8-18.082-4.1-17.382-3.3Z"/><path fill="#442823" d="M99.204 34.9s13.687 17.5 13.487 39.3l5.495-11.2c-.1 0-4.896-14.3-18.982-28.1z"/><path fill="#56332b" d="M39.062 28.9s-11.588 9.7-14.985 20.9C20.68 61 22.079 74.6 19.48 76.3c-3.696 2.4-7.892-11.9-9.29-13.4C7.991 60.5.7 54.4.2 53.3c-.5-1.1 14.786-17.8 21.48-21.1 8.091-3.8 18.082-4.1 17.382-3.3Z"/><path fill="#442823" d="M29.172 34.9S15.485 52.4 15.685 74.2L10.29 63s4.796-14.3 18.882-28.1z"/><path fill="#ccac8d" d="M21.78 74.6s.998 5.4 2.597 7.1c1.598 1.7.5-1.3.5-1.3s-1.7-.9-1.4-7.8c.3-6.9-1.698 2-1.698 2zm85.217 0s-1 5.4-2.598 7.1c-1.598 1.7-.5-1.3-.5-1.3s1.699-.9 1.4-7.8c-.3-6.9 1.698 2 1.698 2z"/><ellipse cx="54.447" cy="70.5" fill="#442823" rx=".799" ry=".8"/><ellipse cx="49.852" cy="75.3" fill="#442823" rx=".799" ry=".8"/><ellipse cx="48.353" cy="70.5" fill="#442823" rx=".799" ry=".8"/><ellipse cx="73.929" cy="70.5" fill="#442823" rx=".799" ry=".8"/><ellipse cx="78.524" cy="75.3" fill="#442823" rx=".799" ry=".8"/><ellipse cx="80.023" cy="70.5" fill="#442823" rx=".799" ry=".8"/></svg>
```